src/schemas/CouponBaseInput.yaml

```diff
@@ -0,0 +1,99 @@
+type: object
+properties:
+  name:
+    type: string
+    description: The name of the coupon.
+    example: 'Startup Deal'
+  code:
+    type: string
+    description: Unique code used to identify the coupon.
+    example: 'startup_deal'
+  description:
+    type: string
+    description: Description of the coupon.
```

Comment:
@ansmonjol 
Could you add `nullable: true` since the description allow the value to be null